Lib/glyphsLib/builder/custom_params.py

```diff
@@ -501,6 +501,32 @@ def to_ufo(self, builder, glyphs, ufo):
 
         ufo.set_info_value("openTypeOS2CodePageRanges", sorted(ufo_codepage_bits))
 
+    def _convert_to_bits(self, codepages):
```

Comment:
this could be decorated as a `@staticmethod` perhaps since it is not using `self`